apps/frontend/src/components/global/checklist/ChecklistRuleInfoHeader.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-card width="100%" class="overflow-y-auto">
+    <v-card-text class="text-center">
+      <h3 class="d-inline-block text-truncate mx-2" style="max-width: 100%">
+        {{ selectedRule.stigRef }}
```

Comment:
should have a default text for when no checklists are selected that says something like "No checklist selected" to match the "No rule selected" text in the section below